addons/kommander/1.5/kommander.yaml

```diff
@@ -9,8 +9,8 @@ metadata:
     # This was originally added to support the PVC's needed for the Kubecost subcomponent.
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.5.0-1"
-    appversion.kubeaddons.mesosphere.io/kommander: "1.5.0"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.5.0-2"
+    appversion.kubeaddons.mesosphere.io/kommander: "1.5.0-beta.0"
```

Comment:
we may not end up releasing 1.5 (nor Konvoy 1.9)

Reply:
yeah.. I thought that might be the case. maybe we can just leave master at the 1.4 release branch so we don't have to backport everything back to 1.4.x 🤔 until we decide 1.5 is gonna be cut